source/includes/databases-collections/databases-collections.scala

```diff
@@ -0,0 +1,103 @@
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+import java.util.concurrent.TimeUnit
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+import scala.jdk.CollectionConverters.SeqHasAsJava
+
+object DatabasesCollections {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+
+    // start-access-database
+    val database = mongoClient.getDatabase("test_database")
+    // end-access-database
+
+    {
+      // start-access-collection
+      val collection = database.getCollection("test_collection")
+      // end-access-collection
+    }
+
+    // start-create-collection
+    val createObservable = database.createCollection("example_collection")
+    Await.result(createObservable.toFuture(), Duration(10, TimeUnit.SECONDS))
+    // end-create-collection
+
+    {
+      // start-find-collections
+      val results = Await.result(database.listCollections().toFuture(), Duration(10, TimeUnit.SECONDS))
+      results.foreach(println)
+      // end-find-collections
+    }
+
+    {
+      // start-find-collection-names
+      val results = Await.result(database.listCollectionNames().toFuture(), Duration(10, TimeUnit.SECONDS))
+      results.foreach(println)
+      // end-find-collection-names
+    }
+
+    {
+      // start-delete-collection
+      val deleteObservable = database.getCollection("test_collection").drop()
+      Await.result(deleteObservable.toFuture(), Duration(10, TimeUnit.SECONDS))
+      // end-delete-collection
+    }
+
+    // start-database-read-prefs
+    val databaseWithReadPrefs =
+        mongoClient.getDatabase("test_database").withReadPreference(ReadPreference.secondary())
+    // end-database-read-prefs
+
+    // start-collection-read-prefs
+    val collectionWithReadPrefs =
+        database.getCollection("test_collection").withReadPreference(ReadPreference.secondary())
+    // end-collection-read-prefs
+
+
+    {
+      // start-tags
+      val tag1 = new Tag("dc", "ny")
+      val tag2 = new Tag("dc", "sf")
+      val tagSet = new TagSet(List(tag1, tag2).asJava)
+
+      val connectionString = ConnectionString("<connection string URI>")
+      val readPreference = ReadPreference.primaryPreferred(tagSet)
+
+      val mongoClientSettings = MongoClientSettings.builder()
+        .applyConnectionString(connectionString)
+        .readPreference(readPreference)
+        .build()
+
+      val clientWithTags = MongoClient(mongoClientSettings)
+      // end-tags
+    }
+
+    {
+      // start-local-threshold
+      val connectionString = ConnectionString("mongodb://localhost:27017")
+
+      val clusterSettings = ClusterSettings.builder()
+        .localThreshold(35, TimeUnit.MILLISECONDS)
+        .build()
+
+      val mongoClientSettings = MongoClientSettings.builder()
+        .applyConnectionString(connectionString)
+        .applyToClusterSettings(builder => builder.applySettings(clusterSettings))
```

Comment:
This is not the correct idiom for overriding a single setting.  Rather it should be:

```
.applyToClusterSettings(builder => builder.localThreshold(35, TimeUnit.MILLISECONDS)
```

The way it is now, it will wipe out any ClusterSetting-related option in the connection string, including the list of hosts.

We should do a sweep through all of our existing docs to check for this.